apps/geo/views.py

```diff
@@ -96,6 +99,23 @@ def get_intersects(self, request, pk=None, version=None):
             for id, geom in geoms
         ])
 
+    @action(
+        detail=True,
+        url_path='published',
+        methods=['post'],
+        serializer_class = RegionSerializer,
```

Comment:
Flake8, please check.

```suggestion
        serializer_class=RegionSerializer,
```